### ISSUE TITLE
Failed install - base table not found

### ISSUE THREAD

**Issue by craigb88** [thedevdojo/voyager#2500]
- Laravel Version: 5.5
- Voyager Version: Latest
- PHP Version: 7.1.13
- Database Driver & Version: MYSQL

### Description:

Install failing on a fresh install of laravel. I have run 

`$ composer require tcg/voyager`

Change my env file to 

```
APP_URL=http://my-domain.dev
DB_HOST=127.0.0.1
DB_DATABASE=fishtank_hub
DB_USERNAME=root
DB_PASSWORD=
```
I've tried with localhost and 127.0.0.1 but every time I run 

`php artisan voyager:install --with-dummy`

I get the following in terminal 

```
Setting up the hooks
Copied File [/vendor/larapack/hooks/publishable/config/hooks.php] To [/config/hooks.php]
Publishing complete.
Hooks are now ready to use! Go ahead and try to "php artisan hook:install test-hook"

In Connection.php line 664:
                                                                                                        
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'fishtank_hub.menus' doesn't exist (SQL: s  
  elect * from `menus` where `name` = admin limit 1)                                                    
                                                                                                        

In PDOConnection.php line 79:
                                                                                                
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'fishtank_hub.menus' doesn't exist  
                                                                                                

In PDOConnection.php line 77:
                                                                                                
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'fishtank_hub.menus' doesn't exist 
```

**Comment by emptynick** [comment]
Please read
https://github.com/the-control-group/voyager/issues/2499

**Comment by fletch3555** [comment]
Will be fixed in v1.0.14, which I'll be releasing later today.
